Demo/Application/Features/Fraud Protection - BTDataCollector/BraintreeDemoBTDataCollectorViewController.m

```diff
@@ -75,31 +67,12 @@ - (void)viewDidLoad {
 
 - (IBAction)tappedCollect {
     self.progressBlock(@"Started collecting all data...");
-    [self.dataCollector collectDeviceData:^(NSString * _Nonnull deviceData) {
+    [self.dataCollector collectDeviceDataWithKountMerchantID:@"60001" :^(NSString * _Nullable deviceData, NSError * _Nullable error) {
```

Comment:
nit: Could we go for a constant here for `60001` even though it's in the Demo app?

Reply:
Sure, it's only used in this 1 place but no harm in extracting it out I guess!